python/ribasim_testmodels/ribasim_testmodels/basic.py

```diff
@@ -459,3 +459,113 @@ def cyclic_time_model() -> Model:
     model.edge.add(fb, bsn)
 
     return model
+
+
+def drought_model() -> Model:
+    """Create a small subsection of the LHM Vecht model containing a drought situation."""
```

Comment:
Drought is a large concept, what do you mean specifically in Ribasim model terms? Would be good to add that to the docstring.